tests/precompile/precompile_test.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2019-2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+// e2e implements the e2e tests.
+package precompile
+
+import (
+	"context"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/ava-labs/avalanchego/api/health"
+	"github.com/ava-labs/subnet-evm/tests/utils"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/go-cmd/cmd"
+	ginkgo "github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+
+	_ "github.com/ava-labs/subnet-evm/tests/precompile/solidity"
+)
+
+var startCmd *cmd.Cmd
+
+func TestE2E(t *testing.T) {
+	gomega.RegisterFailHandler(ginkgo.Fail)
+	ginkgo.RunSpecs(t, "subnet-evm e2e test suites")
+}
+
+// BeforeSuite starts an AvalancheGo process to use for the e2e tests
+var _ = ginkgo.BeforeSuite(func() {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
+	defer cancel()
+
+	var err error
+	wd, err := os.Getwd()
+	gomega.Expect(err).Should(gomega.BeNil())
+	log.Info("Starting AvalancheGo node", "wd", wd)
```

Comment:
will these logs get printed? if so can we switch all logs to geth style?

Reply:
Yes these logs get printed. I think this can be a separate PR since this is not a change from the previous code, which was using different loggers.